internal/cli/performanceadvisor/suggestedindexes/list.go

```diff
@@ -108,5 +111,8 @@ func ListBuilder() *cobra.Command {
 
 	cmd.Flags().StringVar(&opts.ProjectID, flag.ProjectID, "", usage.ProjectID)
 	cmd.Flags().StringVarP(&opts.Output, flag.Output, flag.OutputShort, "", usage.FormatOut)
+
+	_ = cmd.MarkFlagRequired(flag.ProcessName)
```

Comment:
```suggestion
```
This not true for cloud manager/ops manager and we reuse the command

Reply:
check https://github.com/mongodb/mongodb-atlas-cli/blob/4255220f79167ec6438d3dbe01118cc5aecdb3d7/internal/cli/performance_advisor_opts.go#L46-L53

Reply:
Will remove this where I added it, thanks